src/app/common/config/config.ts

```diff
@@ -188,6 +188,7 @@ const config = {
   ONBOARD: {
     API_KEY: process.env.REACT_APP_BLOCKNATIVE_KEY,
     NETWORK_ID: networkId,
+    PROJECT_ID: 'c93804911b583e5cacf856eee58655e6',
```

Comment:
where does this id string come from?

Reply:
@axelrod-blox I edited the question probably you didn't see. Do we have different id's for different env? if so lets move it to .env